java/android/nnstreamer/build.gradle

```diff
@@ -1,16 +1,17 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion '28.0.3'
+    namespace "org.nnsuite.nnstreamer"
+    compileSdk 34
     defaultConfig {
         minSdkVersion 24
-        targetSdkVersion 28
+        targetSdkVersion 34
```

Comment:
Question)
The reason why `targetSdkVersion` is 34 is that it is minimal required version of Google Play Store. right?

Reply:
Yes we should use 34 version for Play Store.